### DISCUSSION TITLE
Cerrah ve eldivenleri sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#66] [Zeka Sorusu] [answered]
Bir cerrah arka arkaya 3 hastayı ameliyat edecektir. Ancak sadece 2 çift eldiveni vardır. Ameliyatların güvenli olması için cerrah eldivenleri kullanmada nasıl bir yol izlemelidir?
Klinik prosedür şöyledir:
a. Doktorun elinin değdiği herhangi bir eldiven yüzeyi hastalara değmeyecektir.
b. Hastalardan herhangi birine değen eldiven yüzeyi doktorun eline veya başka bir hastaya değmeyecektir.
c. Her hastada iki elini de kullanacaktır.

**Comment by ArvenAA** [comment, marked as answer]
İki hastada eldivenleri normal bir şekilde kullanırken, üçüncü hastada sağ eldiveni sol ele, sol eldiveni ise sağ ele takar. Eldivenlerin üst yüzeyleri alta gelecek şekilde, alt yüzeyleri ise üste gelecek şekilde çevrilmelidir.
